includes/utils.h

```diff
@@ -34,7 +34,7 @@ void			free_redirs(t_redir *redirs, int nb_redirs);
 
 /* Error and/or Exit utils */
 t_error			error_and_exit(t_error code);
-t_error	        error_and_return(t_error code, int exit_code);
+int 	        error_and_return(t_error code, int exit_code);

```

Comment:
oh can you git pull from origin master ?
I mean i know those t_error are not necessary for the code to work but they're nice to know what type of value to expect =)

Reply:
(I changed those in my last little cleanup merge)

Reply:
Unless you changed it back to int on purpose :o

Reply:
I did 😅 cos that actually wasn't supposed to return a t_error, was really supposed to return an int. Most of the times I'm using it I'm returning 1, which is really supposed to be 1, not a t_error. And I do this cos what I return is usually the exit_code, and I store it and need it as such, which has no correspondance with our own homemade t_errors

Reply:
ok !